Test/git-issues/git-issue-2380.dfy

```diff
@@ -0,0 +1,15 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method M(t: map<int, int>, k: int)
+  requires k in t
+{
+  assert |t| == |t - {k}| + 1;
```

Comment:
Could you assert something about the cardinality of the values and of the items? That way, you can also test your two other axioms

Reply:
Good idea!  Done.